Check for leap year

```diff
@@ -0,0 +1,28 @@
+# Python program to check if year is a leap year or not
+
+year = 2000
+
+# To get year (integer input) from the user
+
+# year = int(input("Enter a year: "))
```

Comment:
* Dont Trust user's Input to be in a certian format. Remove Trailing whitespaces, and handle input & conversion errors.
* This code is commented out and instead `year` is predefined above, this can't be changed on merging. Do make sure that the commit is up to date and is the final version of your code, not just for testing reason.